features/user_can_delete_post.feature

```diff
@@ -0,0 +1,20 @@
+Feature:
+"As a user,
+In order to get rid of a wrong post,
+I would like to able to delete it."
+
+  Background:
+    Given the following post exists
+    | title        | text          | 
+    | Codeacademy  | Smooth-ride   | 
+    And the following user exists
+    | name    | email            | password     |
+    | Fat Bob | fatbob@gmail.com | ilovecatfood |
+    And I am logged in as "Fat Bob"  
+
+  Scenario: Reader can open up posts
```

Comment:
You should have a scenario where another user is logged in and cant see the delete button

Reply:
Test created and passing